main.js

```diff
@@ -1,72 +1,84 @@
+/* eslint max-classes-per-file: ["error", 3] */
 const form = document.querySelector('#form');
 const bookContainer = document.getElementById('book-container');
-
 let library = [];
-function getBooks() {
-  if (localStorage.getItem('library')) {
-    library = JSON.parse(localStorage.getItem('library'));
-  } else {
-    library = [];
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
```

Comment:
- [ ] The class Book should have at least four methods: `constructor`, `addBook`, `removeBook`, `getBooks`